docs/applications/cli.md

```diff
@@ -0,0 +1,59 @@
+---
+id: cli
+title: bitcoin-s cli
+---
+
+
+## Bitcoin-s command line interface
+
+The [cli](../../app/cli/) project is meant to be a bitcoin-s command line interface (cli). It uses [graalvm native image](https://www.graalvm.org/docs/reference-manual/native-image/) to
+create a native executable that circumvents the jvm for fast start up time.
+
+### Building the command line interface
+
+#### Installing graalvm native image
+First to build the command line interface you need to have the graal jdk installed. This can be installed on [graalvm's github](https://github.com/graalvm/graalvm-ce-builds/releases/tag/vm-19.3.0).
```

Comment:
The link shouldn't point to a specific tag, can just do `https://github.com/graalvm/graalvm-ce-builds/releases/`